src/state/pools/fetchPools.ts

```diff
@@ -23,12 +23,13 @@ export const fetchPoolsBlockLimits = async () => {
     }
   })
 
-  const starts = await multicall(sousChefABI, callsStartBlock)
-  const ends = await multicall(sousChefABI, callsEndBlock)
+  // commented out for testnet
+  // const starts = await multicall(sousChefABI, callsStartBlock)
+  // const ends = await multicall(sousChefABI, callsEndBlock)
 
   return poolsWithEnd.map((cakePoolConfig, index) => {
-    const startBlock = starts[index]
-    const endBlock = ends[index]
+    const startBlock = '0x011025c8' // hard coded for testnet
```

Comment:
as much as possible ilaag n alang mga arog kaini sa config. kasi kung nalilingawan nagkakaproblema. you should code in testnet na parang nasa mainnet ka